schemas/groups/navigation.json

```diff
@@ -435,14 +460,31 @@
         }
       }]
     },
+    "manouver": {
+			"type": "object",
+			"description": "Special maneuver such as regional passing arrangement. (from ais)",
+			"allOf": [{
+        "$ref": "../definitions.json#/definitions/commonValueFields"
+      }],
+			"properties": {
+				"value": {
+					"type": "string",
+					"enum": [
+						"Not available (default)",
```

Comment:
I think the (default) is left over from a cut and paste from the AIS spec which states that this is the default setting when a Class A transponder is turned ON.

From Signal K point of view we will just show "Not Available" if this is what we have received from the AIS.